docs/native/src/onboarding/onboarding.rst

```diff
@@ -22,67 +22,72 @@ Onboarding information
 The following information must be supplied to Microsoft to enable end-to-end testing and release of
 |Office iOS| integration.
 
-+----------------+-------------+--------------------------------------------+--------------------------------------------------+
-| Property       | Data Type   | Sample Value                               | Description                                      |
++-----------------+-------------+--------------------------------------------+--------------------------------------------------+
+| Property        | Data Type   | Sample Value                               | Description                                      |
```

Comment:
When I go to "View", I don't see your table and I think that's because you added a '-' to this row, but you didn't extend all of the rows in this column with a '-'.